.github/workflows/super-linter.yml

```diff
@@ -26,7 +26,6 @@ jobs:
         env:
           VALIDATE_ALL_CODEBASE: false
           LINTER_RULES_PATH: /
-          VALIDATE_JAVASCRIPT_STANDARD: false
```

Comment:
Đừng sửa file này.

Reply:
Cái dòng đó lúc chạy check bị lỗi "Behavior not supported, please either only include (VALIDATE=true) or exclude (VALIDATE=false) linters, but not both"

Reply:
Chuẩn rồi. Nãy cũng mới xem lại sửa lỗi đó https://github.com/lelinhtinh/Userscript/commit/bd63d232f595b159f9d4b3b196790f93bae112ad